### PR TITLE
Fix: score_els has no method foreach,

### PR DESCRIPTION
**Pull request by xkbeyer**
In edge the missing method results in an exception.

### CODE REVIEW

**Review by ThibaultJanBeyer — APPROVED** [review]
looks gooooooowdd

File: lib/main.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ function mainloop() {
     document.title = `Longcat is lo${'o'.repeat(the_physics.score)}ng!`
 
     // Update the score itself.
-    score_els.forEach(el => { el.innerHTML = the_physics.score.toFixed(1); })
+    score_els.innerHTML = the_physics.score.toFixed(1);
```
Comment (ThibaultJanBeyer) [inline lib/main.js:54]: well done

File: lib/main.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ window.requestAnimationFrame = window.requestAnimationFrame
                             || window.msRequestAnimationFrame
                             || function(cb) { window.setTimeout(cb, 1000/60); };
 
-const score_els = document.querySelectorAll('.score__number');
+const score_els = document.querySelector('.score__number');
```
Comment (ThibaultJanBeyer) [inline lib/main.js:27]: nice

**Review by ThibaultJanBeyer — APPROVED** [review]
nice

**Review by lucasb-eyer — APPROVED** [review]
LGTM

**Pull request merged**